misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -240,6 +241,25 @@ class JettyService @Inject internal constructor(
     // TODO(mmihic): Force security handler?
     val servletContextHandler = ServletContextHandler()
     servletContextHandler.addServlet(ServletHolder(webActionsServlet), "/*")
+//
+//    val inflaterPool = InflaterPool(333, false)
+//    val deflaterPool = DeflaterPool(333, Deflater.BEST_SPEED, false)
+//    servletContextHandler.setAttribute(
+//      WebSocketServerComponents.WEBSOCKET_DEFLATER_POOL_ATTRIBUTE,
+//      deflaterPool
+//    )
+//    servletContextHandler.setAttribute(
+//      WebSocketServerComponents.WEBSOCKET_INFLATER_POOL_ATTRIBUTE,
+//      inflaterPool
+//    )
+//
+
+//    servletContextHandler.addServletContainerInitializer(ServletContainerInitializerHolder({ c, ctx->
+//      WebSocketServerComponents.ensureWebSocketComponents(server, servletContext)
+//      JettyWebSocketServerContainer.ensureContainer(servletContext)
+//    }))
```

Comment:
These are possible config we could use to tune websockets behavior. However, I just went with the default values for now.